docs/sources/userguide/plugins.md

```diff
@@ -0,0 +1,157 @@
+Docker Plugins MVP
+==================
+
+Plugins provide a mechanism for hooking into the Docker engine to extend its behavior.
+
+HTTP over a local UNIX socket is used to communicate between Docker and its plugins.
+
+Plugins are distributed as containers.
+
+This is targeted at Docker 1.7.
+
+# Usage
+
+Plugins can be loaded using a special `docker plugin load` command, as follows:
+
+```
+$ docker plugin load -e $ARGS clusterhq/flocker-plugin $@
```

Comment:
The `-e $ARGS` was for environment variables to pass to the plugin, and `$@` just to say that you can pass some arguments. Should we just pick one of these? Or have neither?

Having config as part of the plugin image doesn't make sense because it's config provided by the cluster admin not the plugin author.